wiki/Tournaments/Template/en.md

```diff
@@ -5,54 +5,60 @@ tags:
 ---
 <!--
 find-replace these curly braces then delete this comment
+{TOURNAMENT_NAME} = tournament name
 {MODE} = game mode
-{ABBR} = abbreviation of the title e.g. MWC
+{ABBR} = abbreviation of the title i.e. MWC
 {XX} = year, last two digits
 {NTH} = installment ordinal number
 {TOPIC_NUM} = forum thread number
 {STATS_LINK} = link to overall statistics (should be Google Drive link)
 you will still to add other items such as tables
 -->
 
-# {MODE} World Cup 20{XX}
+# {TOURNAMENT_NAME} World Cup 20{XX}
```

Comment:
`{TOURNAMENT_NAME} World Cup 20{XX}` -> `{TOURNAMENT_NAME}`

Are you formatting this for community tournaments too? If so, do this change and fix other instances.